lib/workers/repository/model/commit-message.ts

```diff
@@ -1,54 +1,87 @@
-export class CommitMessage {
-  public static readonly SEPARATOR: string = ':';
+import type { CommitMessageJSON } from '../../../types';
 
-  private message = '';
+/**
+ * @see https://git-scm.com/docs/git-commit#_discussion
+ *
+ * [optional prefix]: <suject>
+ * [optional body]
+ * [optional footer]
+ */
+export abstract class CommitMessage {
+  static readonly #SEPARATOR: string = ':';
```

Comment:
please revert to use typescript private declaration, as node private fields have performance regressions until node v18, wich is far away for renovate.

Reply:
Wow, interesting. Changes are done. Could you please provide a link with more information?

Reply:
https://nodejs.org/en/blog/release/v18.0.0/#v8-10-1